cmd/zt_traverser_managed_disk_test.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import "testing"
+
+func TestLargeManagedDiskSnapshot(t *testing.T) {
```

Comment:
technically the issue is "covered" by the other test, but I wonder if we can't mock just the single request or two that'd get made by the HTTP client under the hood here.

Reply:
GetProperties request/responses are mocked here cmd/zt_traverser_blob_test.go!